crawl-ref/source/monster.cc

```diff
@@ -5868,7 +5868,12 @@ reach_type monster::reach_range() const
     {
         const mon_attack_def attk(mons_attack_spec(*this, i));
         if (flavour_has_reach(attk.flavour) && attk.damage)
+        {
+        if (!flavour_has_disto(attk.flavour))
             range = REACH_TWO;
+        else
+            range = REACH_THREE;
+        }
```

Comment:
Indentation here is not correct. 

Also, I think for future proofing you should rename the af to `AF_RIFT` and explicitly give only `AF_RIFT` reach three instead of implementing this nested check. (Though AF_RIFT should count as reaching for the mon_util function).

Reply:
having minor difficulty writing this "correctly" but yeah, dig it!